src/components/Pricing.vue

```diff
@@ -0,0 +1,72 @@
+<template>
```

Comment:
All this file can be deleted. We can just explain the quotas and pricing in the FAQ below

Reply:
done